src/software/thunderscope/replay/BUILD

```diff
@@ -34,3 +34,14 @@ py_library(
         ":proto_player",
     ],
 )
+
+py_binary(
+    name = "replay_file_debugging_script",
+    srcs = [
+        "replay_constants.py",
```

Comment:
what is replay_constants.py?

Reply:
if you do add a `constants` file, you can call it `constants.py` and add it as a module to this BUILD

Reply:
this is already in master. I just merged stuff that are from here: https://github.com/nimazareian/Software/commit/17bd68131cd3402d50ee32665fb525e03be44fc8

Reply:
Yeah that file is pretty unnecessary, but it's been in the codebase since the introduction of replays so I decided to just leave it in my PR.